### DISCUSSION TITLE
Aparecen las tareas una vez que son borradas y se actualiza el navegador

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#328] [4. Bug Reports] [answered]
Hola

Resulta que cuando borro las tareas, me aparece el mensaje que fue eliminado, pero cuando actualizo la página aparecen nuevamente. El usuario tiene la opción de "**editor**".

Adjunto foto y video
![image](https://user-images.githubusercontent.com/61419471/124939590-3c720480-dfd7-11eb-899f-71488e0389f2.png)

**Link video**: https://drive.google.com/file/d/1LmuAhlurwYbERAld54lB6R4zQFr7Dw4K/view?usp=sharing

Gracias =>

**Comment by acedward** [comment, marked as answer]
El día lunes hay un update que probablemente corrige esto.
Gracias por el reporte

**Reply by NicolasPLG** [reply]
hola @acedward muchas gracias por la información. La próxima semana lo revisaré.
